invisible_cities/cities/components_test.py

```diff
@@ -487,6 +489,44 @@ def test_read_wrong_pmt_ids(ICDATADIR):
         next(sns_gen)
 
 
+@mark.parametrize("apply_z", (True, False))
+def test_hits_corrector_z_uncorrected_when_flagged( correction_map_filename
+                                                  , apply_z):
+    '''
+    Test that checks to ensure that z is uncorrected when `apply_z` is True
+    '''
+    apply_temp = False
+    norm_strat = NormStrategy.kr
+    norm_value = None
```

Comment:
Since these are only used in the call to hits corrector, pass them directly as keyword arguments to reduce noise